### ISSUE TITLE
2005 ipcr not parsed **lower priority**

### ISSUE THREAD

**Issue by laironald** [funginstitute/patentprocessor#18]
2005 ipcr doesn't appear to be parsed.
see stats indicated in the [Google Spreadsheet](https://docs.google.com/spreadsheet/ccc?key=0AoSg8Kx6M9EBdGt0TUJVWms3bHJFZi04NXNBVUlrMmc#gid=0)

**Comment by laironald** [comment]
**might not be a bug. but a reason for why?

**Comment by gtfierro** [comment]
I think it's a difference in the XML, but I'm not sure.

**Comment by laironald** [comment]
right. we may want to address this issue when the dust settles a bit

On Thursday, August 22, 2013, Gabe Fierro wrote:

> I think it's a difference in the XML, but I'm not sure.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/funginstitute/patentprocessor/issues/18#issuecomment-23117062
> .

## 

Sent from mobile
